vue-components/src/components/Editor.js

```diff
@@ -112,9 +112,10 @@ export default {
       provider.injectCSS();
     }
 
-    this.editor.onDidChangeModelContent(() =>
-      this.$emit("input", this.editor.getValue())
-    );
+    this.editor.onDidChangeModelContent(() => {
+      this.$emit("update:modelValue", this.editor.getValue());
+      this.$emit("input", this.editor.getValue());
```

Comment:
A solution for the double `input` event call would be to remove it in favor of just using the `modeValue`?

Reply:
Otherwise, I could wrap in this:

```
      const newValue = this.editor.getValue();
      if (this.modelValue !== newValue) {
          this.$emit("update:modelValue", this.editor.getValue());
          this.$emit("input", this.editor.getValue());
      }
```

Reply:
To skip the double update, just keep the last sent value in a local var and just skip the emit if same as before.

```
this.editor.onDidChangeModelContent(() => {
      const newValue = this.editor.getValue();
      if (this.lastValue === newValue) {
            return;
      }
      this.lastValue = newValue;
      this.$emit("update:modelValue", newValue);
      this.$emit("input", newValue);
}
```

Reply:
This still seems to occur...

I set `input="console.log($event)"`, and in the console it first prints the contents then and `InputEvent`.